dask_ml/preprocessing/_encoders.py

```diff
@@ -116,22 +116,29 @@ def __init__(
         categorical_features: Optional[pd.Categorical] = None,
         categories: Union[str, ArrayLike] = "auto",
         drop: Optional[bool] = None,
-        sparse: bool = True,
+        sparse_output: bool = True,
```

Comment:
Should  this renaming of `sparse` to `sparse_output` trigger a deprecation warning?

Reply:
`sklearn.preprocessing.OneHotEncoder` will raise a warning.